Userland/Demos/Mouse/main.cpp

```diff
@@ -127,6 +109,35 @@ class MainFrame final : public GUI::Frame {
             painter.draw_line(p1, p2, Color::Red, 2);
             painter.draw_line(p3, p4, Color::Red, 2);
         }
+
+        // draw mouse outline
+        path.move_to({ 30, 140 });
+        path.line_to({ 30, 65 });
+        path.line_to({ 130, 65 });
+        path.line_to({ 130, 140 });
+        path.line_to({ 30, 140 });
+        painter.stroke_path(path, Color::Black, 1);
+        path.clear();
+
+        // draw fw button
```

Comment:
I get that this is a relocation of the comment but if these comments are to be kept, in your opinion, then I would take the opportunity to write fw as forward.